include/mapfs/mac_00_hit.h

```diff
@@ -1,3 +1,31 @@
+#if VERSION_JP

```

Comment:
I noticed there are some unused values among the colliders (`COLLIDER_Root`, for example). I have no idea where those came from, so I'm just setting the ones that are actually used for JP.

Reply:
they come from the model binary files iirc. yeah, no worries about leaving out unused ones

Reply:
yes, these headers were generated using mapfs assets decoded by Star Rod. we should eventually be generating them at build-time from the content of the _hit and _shape assets, and this will properly account for any differences between versions. for now this is totally fine